src/utils.js

```diff
@@ -229,7 +229,9 @@ export function getUiOptions(uiSchema) {
 }
 
 export function isObject(thing) {
-  return typeof thing === "object" && thing !== null && !Array.isArray(thing);
+  return thing instanceof File
```

Comment:
Can you actually add an if statement here? For readability, and if we need to do additional checks in the future

Reply:
I do that

Reply:
Done